hostprocess/flannel/build.sh

```diff
@@ -0,0 +1,3 @@
+docker buildx create --name img-builder --use --platform windows/amd64
+trap 'docker buildx rm img-builder' EXIT
+docker buildx build --platform windows/amd64 --output=type=registry --pull -f Dockerfile -t jsturtevant/flannel:hostprocess .
```

Comment:
Let's not hardcode your dockerhub repo :)

Reply:
agree, before I pushed to the sig-windows-tools repo I wanted to get general agreement in this direction.

Reply:
fixed